setup/Swix/Microsoft.FSharp.Dependencies/Dependencies.swr

```diff
@@ -0,0 +1,43 @@
+use vs
+
+package name=Microsoft.FSharp.Dependencies
+        version=4.1
```

Comment:
This line can be replaced with `version=$(Version)` if you add that variable to the `.swixproj` below (I'll mark it) and that would make changing version numbers easier in the future.

Similarly for all other `version=4.1` lines in this file.